### PR TITLE
[DOCS] Add tslint-microsoft-contrib to roadmap

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -10,168 +10,305 @@
...
+| tslint-microsoft-contrib rule         |     | ESLint rule                                      |
+| ------------------------------------- | :-: | ------------------------------------------------ |
+| `export-name`                         | TBD | ?                                                |
+| `function-name`                       | TBD | ?                                                |
+| `import-name`                         | TBD | ?                                                |
+| `function-name`                       | TBD | ?                                                |
+| `informative-docs`                    | TBD | ?                                                |
+| `insecure-random`                     | 🔌  | [custom implementation][insecure-random]         |
+| `max-func-body-length`                | 🌟  | [`max-statements`][max-statements]               |
+| `missing-jsdoc`                       | TBD | [`max-statements`][max-statements]               |
```
Comment (bradzacher) [inline ROADMAP.md]: wrong rule

**Comment by j-f1** [comment]
This is ready for another review.

**Review by bradzacher — APPROVED** [review]

File: ROADMAP.md [hunk 2]
```diff
@@ -6,172 +6,312 @@
 🌟 = in ESLint core  
 🔌 = in another plugin  
 🛑 = unimplemented  
-❓ = implementations differ or ESLint version is missing functionality
+◐ = implementations differ or ESLint version is missing functionality
```
Comment (armano2) [inline ROADMAP.md]: what do you think about using here: 🌗 ?

Comment (j-f1) [reply ROADMAP.md]: Nice one!

Comment (armano2) [reply ROADMAP.md]: there are 2 variants 👍 

🌓 and 🌗

**Pull request merged**